### PR TITLE
API

### PR DESCRIPTION
**Pull request by AlyAtalla**
Implemented the form "Submit" button that will save a score for a game created by you (sending data to the API).
Used arrow functions instead of the function keyword.
Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/modules/Get.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const id = 'ZTp6WjEdBYQQYfIGvFXP';
+export const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores`;
+
+export default async function getUsers() {
```
Comment (fobadara) [inline src/modules/Get.js]: [REQUIRED] In this milestone it is a requirement to make use of arrow functions. Kindly refactor your code to satisfy this condition.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @AlyAtalla, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good first attempt
✅ Use of async and await

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/Get.js (hunk 1 above)
Comment (AlyAtalla) [reply src/modules/Get.js]: thank you.

**Review by danmainah — APPROVED** [review]
Hi @AlyAtalla , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Implemented change requested to ensure all methods are using arrow function format as per the project requirement.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**